src/uct/cuda/cuda_iface.c

```diff
@@ -73,7 +75,7 @@ static ucs_status_t uct_cuda_iface_query(uct_iface_h iface,
 static uct_iface_ops_t uct_cuda_iface_ops = {
     .iface_close         = UCS_CLASS_DELETE_FUNC_NAME(uct_cuda_iface_t),
     .iface_get_address   = uct_cuda_iface_get_address,
-    .iface_get_device_address = (void*)ucs_empty_function_return_success,
+    .iface_get_device_address = uct_sm_iface_get_device_address,
```

Comment:
you should not reference uct_sm_iface_get_device_address sm function from cuda tl